static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -365,6 +365,7 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
           window.scrollTo(0, 0);
           this._container.classList.toggle('VerticalFullPageMap--listShown');
           this._container.classList.toggle('VerticalFullPageMap--mapShown');
+          this._pageWrapperEl.classList.toggle('YxtPage-wrapper--mapShown');
```

Comment:
Is `YxtPage` a new selector prefix that was added as part of the locator work? It sounds like this class is added to the `div` that wraps the entire page. I'm wondering if we should refactor some of our existing CSS to use this prefix as well. If we have styling that applies to any page generated by the Theme, it seems like we could use this selector prefix with it. Out of scope of this item! Just an observation.

Reply:
Definitely agree! Was here before locator work: https://github.com/yext/answers-hitchhiker-theme/blob/master/layouts/html.hbs#L153